2200110/230120_test_06.py

```diff
@@ -0,0 +1,30 @@
+import collections
+from collections import Counter
+
+# 아래 풀이는 vscode에서 잘 되는데 swea 제출에서 계속 fail ...
+# 이유가 뭘까?
+
+T = int(input())
+for t in range(1, T+1):
+    my_dict = dict(collections.Counter(list(map(int, input().split()))))
+    many = sorted(my_dict.items(), key=lambda x: x[0])[-1]
```

Comment:
```suggestion
    many = sorted(my_dict.items(), key=lambda x: x[1])[-1]
```
갯수에 대해 정렬을 하려면 위와 같이 수정해주세요!